turingschool-examples/flashcards-starter
# Feature/iteration1

**What does this PR do?**
- Completes iteration 1
- Provides testing suites for Card and Turn classes
- Builds functioning Card and Turn classes that pass all unit tests
- Adds other necessary file architecture for later iterations

**Is this a feature or refactor?**
- Feature

**Problems Encountered & Resulting Solutions**
- None

**State any problems encountered**
- None

**Where should the reviewer start?**
- new classes can be found in `src/Card.js` and `src/Turn.js`
- corresponding unit tests can be found in `test/Card-test.js` and `test/Turn-test.js`

**How should this be tested?**
- Run `npm test` on the repository to verify that all tests are being passed for these two classes

**Steps To Take**
- Continue to iteration 2, build tests and continue building data model.
